onnxruntime/test/python/transformers/test_optimizer_utils.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
nit: lint

Reply:
My local lint doesn't catch this

Reply:
Fixed. Seems lint can only work after the file was "modified".